reckruitment/index.php

```diff
@@ -35,47 +37,66 @@
+<!--<p>The 2020 Ramblin’ Reck Club application is now closed. Check back next December for 2021's application!</p>-->
+    <p>The 2019 Ramblin’ Reck Club application can be found <a href="https://www.google.com/url?q=https://forms.gle/dazCjaZ2ZW1QWcmaA&sa=D&ust=1577554565568000&usg=AFQjCNFTszg9wFc_DmUw_RzErWikg4F-PA" target="_blank">here</a>. A completed written application must be submitted no later than <strong>Saturday, January 26th at 11:59pm</strong>.</p>
 
-<h4 class="mb-3">Information Sessions (Non-Mandatory)</h4>
+    <br>
+<h4 class="mb-3" id="info-sessions">Information Sessions (Non-Mandatory)</h4>
 
 <p>For anyone interested in learning more about Ramblin’ Reck Club, what we are involved in on campus, opportunities within the Club, and the recruitment process, we will be hosting three non-mandatory information sessions before the application deadline closes. Feel free to stop in and introduce yourself and get the low-down on all things RRC during one of the following sessions:</p>
 
 <ul>
-<!--    <li><strong>Tuesday, January 15</strong>: DM Smith 304 from 6:00pm-7:00pm</li>-->
-<!--    <li><strong>Thursday, January 17</strong>: DM Smith 304 from 6:00pm-7:00pm</li>-->
-<!--    <li><strong>Tuesday, January 22</strong>: DM Smith 304 from 6:00pm-7:00pm</li>-->
+    <li><strong>Tuesday, January 14th</strong>: Skiles 309 from 7:00pm-8:00pm</li>
+    <li><strong>Thursday, January 16th</strong>: Skiles 309 from 7:00pm-8:00pm</li>
+    <li><strong>Tuesday, January 21st</strong>: Skiles 309 from 7:00pm-8:00pm</li>
 <!--    <li><strong>First Social: Monday, January 28th at 7:15pm (Mandatory)</strong></li>-->
-  <li>Check back here in December 2019 for info session dates for the 2020 recruitment cycle!</li>
+<!--  <li>Check back here in December 2019 for info session dates for the 2020 recruitment cycle!</li>-->
 </ul>
-
+<br>
+<h4 class="mb-3" id="first-social">First Social (Mandatory)</h4>
     <p>Attending this social is your chance as an applicant to meet Reck Clubbers for the first time, get a feel for what Ramblin’ Reck Club is all about, ask any questions you may have about the Club or getting involved, and most importantly, sign up for an interview time. No need to stress about this event; be yourself and talk to anyone you do not already know! Dress code for this event is <strong>business casual</strong>, and will be hosted in the Edge Building Trophy Room (on the corner of Techwood and Bobby Dodd).</p>
+    <ul>
+        <li><strong>First Social: Monday, February 3rd</strong></li>
+    </ul>
+    <p>There are two time slots for the first social: <strong>7:00pm-8:00pm and 8:30pm-9:30pm</strong>. Please note down your availability on the application. If you are not available during either of these times, please email <a href="mailto:membershipchair@reckclub.org">Kassie Lee</a>.</p>
 
-<h4 class="mb-3">RRC/Athletics Bonding (Mandatory)</h4>
+<br>
```

Comment:
You may just want to make this a class"mb-2" on the `<p>` tag above it. `<br>` tags could inhibit the responsiveness/resizing of the page of different devices.